flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/common/ExpandTestPrograms.java

```diff
@@ -29,7 +29,7 @@
 /** {@link TableTestProgram} definitions for testing {@link StreamExecExpand}. */
```

Comment:
Update JavaDoc. Also for other program classes?

Reply:
I'm fine with fixing it in the same PR. It should be a rather small change.

Reply:
So far I've opted for just this one change. 

I'd prefer to fix things more broadly in a later PR.